rules/sinks/third_parties/sdk/amazonaws/java.yaml

```diff
@@ -1014,14 +949,6 @@ sinks:
       - "(?i)(com[.]amazonaws[.]services[.]securitylake).*"
     tags:
 
-  - id: ThirdParties.SDK.Amazonaws.SageMakerGeospatial
-    name: Amazonaws SageMaker Geospatial
-    domains:
```

Comment:
@hiaashna Why have we removed couple of rules here?

Reply:
there is already one rule for `sagemaker` which has the following pattern:
`com[.]amazonaws[.]services[.]sagemaker).*` so it would be detected there. Do we need to to break that sagemaker also into individual services?
and similarly for other services such as `translate`, `forecast`, `personalize` and `ssm`

Reply:
ok, then it makes sense